InAppUtils/InAppUtils.m

```diff
@@ -28,43 +28,30 @@ - (dispatch_queue_t)methodQueue
     return dispatch_get_main_queue();
 }
 
-RCT_EXPORT_MODULE()
+RCT_EXPORT_MODULE();
 
-- (void)paymentQueue:(SKPaymentQueue *)queue
- updatedTransactions:(NSArray *)transactions
+- (void) paymentQueue:(SKPaymentQueue *)queue
+  updatedTransactions:(NSArray *)transactions
 {
     for (SKPaymentTransaction *transaction in transactions) {
         switch (transaction.transactionState) {
             case SKPaymentTransactionStateFailed: {
-                NSString *key = RCTKeyForInstance(transaction.payment.productIdentifier);
+                NSString *key = transaction.payment.productIdentifier;
                 RCTResponseSenderBlock callback = _callbacks[key];
                 if (callback) {
-                    switch(transaction.error.code) {
-                        case SKErrorPaymentCancelled:
-                            callback(@[@"user_cancelled"]);
```

Comment:
this is intentional for when the user cancels out of the IAP flow to distinguish b/t that and real errors; should make sure we're not displaying an error to the user in case they cancel out here.